source/common/upstream/stat_names.h

```diff
@@ -0,0 +1,26 @@
+#include "envoy/stats/stats_macros.h"
+
+#pragma once
+
+namespace Envoy {
+namespace Upstream {
+
+/**
+ * All cluster manager stats. @see stats_macros.h
+ */
+#define ALL_CLUSTER_MANAGER_STATS(COUNTER, GAUGE, HISTOGRAM, TEXT_READOUT, STATNAME)               \
```

Comment:
Per my other comment, I'm pretty skeptical of this one. The CM is created once?

Reply:
I think I had misunderstood the comment. What you are saying here is just that we can leave this code as it was because CMs are not recreated with xDS. Is that right? I thought maybe they were recreated with xDS based on a flame-graph but maybe I misinterpreted.

Reply:
Looked at flame-graph again, and I hacked the wrong set of stats. I should've done ALL_CLUSTER_STATS. will fix.

Reply:
Yeah exactly, sorry for the confusion.